lambda_code/scan/scan.py

```diff
@@ -236,8 +237,7 @@ def cname_s3(account_name, record_sets):
         for r in record_sets
         if r["Type"] in ["CNAME"]
         and "ResourceRecords" in r
-        and "amazonaws.com" in r["ResourceRecords"][0]["Value"]
-        and ".s3-website" in r["ResourceRecords"][0]["Value"]
+        and is_s3_website_endpoint_url(r["ResourceRecords"][0]["Value"])
```

Comment:
It's possible that there may be a period at the end of the DNS record in some cases